src/components/side_panel/chart/chart_with_axis/design_panel.xml

```diff
@@ -29,100 +21,11 @@
-              onChange.bind="updateDataSeriesAxis"
-            />
-          </Section>
-          <Section class="'pt-0 px-0'">
-            <t t-set-slot="title">Series name</t>
-            <input
-              class="o-input o-serie-label-editor"
-              type="text"
-              t-att-value="getDataSerieLabel()"
-              t-on-change="(ev) => this.updateDataSeriesLabel(ev)"
-            />
-          </Section>
-          <Section class="'pt-0 px-0 o-show-trend-line'" t-if="!props.definition.horizontal">
-            <t t-set-slot="title">Trend line</t>
-            <t t-set="showTrendLineLabel">Show trend line</t>
-            <t t-set="trend" t-value="getTrendLineConfiguration()"/>
-            <t t-set="trendType" t-value="getTrendType(trend)"/>
-            <Checkbox
-              name="'showTrendLine'"
-              label="showTrendLineLabel"
-              value="trend !== undefined and trend.display"
-              onChange.bind="toggleDataTrend"
-            />
-            <div t-if="trend !== undefined and trend.display">
-              <div class="d-flex py-2">
-                <div class="w-100">
-                  <span class="o-section-subtitle">Type</span>
-                  <select class="o-input trend-type-selector" t-on-change="this.onChangeTrendType">
-                    <option value="linear" t-att-selected="trendType === 'linear'">Linear</option>
-                    <option value="exponential" t-att-selected="trendType === 'exponential'">
-                      Exponential
-                    </option>
-                    <option value="polynomial" t-att-selected="trendType === 'polynomial'">
-                      Polynomial
-                    </option>
-                    <option value="logarithmic" t-att-selected="trendType === 'logarithmic'">
-                      Logarithmic
-                    </option>
-                  </select>
-                </div>
-                <div class="w-50 ms-3" t-if="trendType === 'polynomial'">
-                  <span class="o-section-subtitle">Degree</span>
-                  <input
-                    t-att-value="trend.order"
-                    type="number"
-                    class="w-100 o-input trend-order-input"
-                    t-on-change="this.onChangePolynomialDegree"
-                    min="1"
-                  />
-                </div>
-              </div>
-              <div class="d-flex align-items-center">
-                <span class="o-section-subtitle my-0 pe-2">Trend line color</span>
-                <RoundColorPicker
-                  currentColor="getTrendLineColor()"
-                  onColorPicked.bind="updateTrendLineColor"
-                />
-              </div>
-            </div>
-          </Section>
-        </Section>
-      </t>
-    </SidePanelCollapsible>
+    <SeriesWithAxisDesignEditor
+      figureId="props.figureId"
+      definition="props.definition"
+      updateChart="props.updateChart"
```

Comment:
`t-props="props"`
same everywhere in this PR.
Existing templates can be updated later or in another commit